### PR TITLE
Hide readme form on validation errors

### PR DESCRIPTION
**Pull request by chenriksson**
-Update bindData to call bindReadMeData when model is null
-Update bindReadMeData to hide form when model is null
-Remove unmatched ending form tag

### CODE REVIEW

File: src/NuGetGallery/Views/Packages/_EditMetadata.cshtml [hunk 1]
```diff
@@ -175,7 +175,7 @@
                     </div>
                 </div>
                 <!-- /ko -->
-            </form>
+            @*</form>*@
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Packages/_EditMetadata.cshtml]: What is this about?

Comment (chenriksson) [reply src/NuGetGallery/Views/Packages/_EditMetadata.cshtml]: When edit support was added, the form was extracted into an _EditForm partial but only the begin tag was removed.

see:
https://github.com/NuGet/NuGetGallery/commit/0434052dba95a1526df5f961bc09ebb9fc5b24f9#diff-5058b8e804fd7b6e9e60a29de7f6fff6L46

Comment (ryuyu) [reply src/NuGetGallery/Views/Packages/_EditMetadata.cshtml]: So should this be removed altogether?

Comment (chenriksson) [reply src/NuGetGallery/Views/Packages/_EditMetadata.cshtml]: Yes - see link above. It should have been removed when the form tags were extracted into _EditForm.

**Pull request merged**